src/layouts/MainLayout.vue

```diff
@@ -1,21 +1,116 @@
-<template>
-  <q-layout view="hHh lpR fFf">
-    <q-page-container>
-      <router-view />
-    </q-page-container>
-  </q-layout>
-</template>
-
 <script lang="ts">
-import { defineComponent } from 'vue';
+// import libraries
+import { setCssVar } from 'quasar';
+import { defineComponent, ref } from 'vue';
+import { i18n } from 'src/boot/i18n';
+
+// import components
+import VueDrawerHeader from 'components/VueDrawerHeader.vue';
+import VueUserSelect from 'components/VueUserSelect.vue';
+import VueDrawerMenu from 'components/VueDrawerMenu.vue';
+
+// import types
+import { User } from 'components/types';
+
+if (window.Cypress) {
+  window.i18n = i18n;
+}
+
+const rideToWorkByBikeConfig: object = JSON.parse(
+  process.env.RIDE_TO_WORK_BY_BIKE_CONFIG
+);
+setCssVar('gray-light', rideToWorkByBikeConfig.colorGrayLight);
 
 export default defineComponent({
   name: 'MainLayout',
 
-  components: {},
+  components: { VueDrawerHeader, VueUserSelect, VueDrawerMenu },
 
   setup() {
-    return {};
+    const leftDrawerOpen = ref(false);
+
+    const users: User[] = [
+      {
+        label: 'User 1',
+        value: '1',
+        image: 'https://picsum.photos/id/40/300/300',
+      },
+      {
+        label: 'User 2',
+        value: '2',
+        image: 'https://picsum.photos/id/64/300/300',
+      },
+      {
+        label: 'User 3',
+        value: '3',
+        image: 'https://picsum.photos/id/91/300/300',
+      },
+    ];
+
+    const toggleLeftDrawer = () => {
+      leftDrawerOpen.value = !leftDrawerOpen.value;
+    };
+
+    return {
+      leftDrawerOpen,
+      users,
+      toggleLeftDrawer,
+    };
   },
 });
 </script>
+
+<template>
+  <q-layout view="hHh lpR fFf">
+    <q-header reveal class="lg-hidden bg-white">
+      <q-toolbar>
+        <vue-drawer-header
+          v-model="leftDrawerOpen"
+          data-cy="drawer-header-mobile"
+          :show-logo="false"
+          :show-drawer-open-button="true"
+        >
+        </vue-drawer-header>
+      </q-toolbar>
+    </q-header>
+
+    <q-drawer
+      show-if-above
+      v-model="leftDrawerOpen"
+      side="left"
+      :width="320"
+      class="bg-gray-light q-py-lg q-px-lg"
+      data-cy="q-drawer"
+    >
+      <vue-drawer-header
+        data-cy="drawer-header"
+        :mobile="false"
+      ></vue-drawer-header>
+      <vue-user-select
+        :options="users"
+        class="q-pt-lg"
+        data-cy="user-select"
+      ></vue-user-select>
+      <vue-drawer-menu class="q-pt-lg" data-cy="drawer-menu"></vue-drawer-menu>
+    </q-drawer>
+
+    <q-page-container>
+      <router-view />
+    </q-page-container>
+
+    <q-footer>
+      <!-- footer content -->
+    </q-footer>
+  </q-layout>
+</template>
+
+<style scoped lang="scss">
+@media (min-width: $breakpoint-md-min) {
+  .lg-hidden {
+    display: none;
+  }
+}
+.bg-gray-light {
```

Comment:
You don't need declare `bg-gray-light` CSS class again. You override default  Quasar CSS class `gray-light` on the line 22 and if you want use it as background color use `bg-gray-light` CSS class.

Reply:
Noted. I originally did not intend to override default Quasar classes, but I understand now that it's a better way.
I also removed custom media-query display class.